src/components/BlockedScenarios/BlockedScenarios.tsx

```diff
@@ -77,16 +104,60 @@ const BlockedScenarios = ({ type }: { type: string }) => {
                 </Text>
             ),
         },
+        p2pBlocked: {
+            actionButton: (
+                <Button onClick={openLiveChat} size='lg' textSize={buttonTextSize}>
+                    <Localize i18n_default_text='Live chat' />
+                </Button>
+            ),
+            description: (
+                <Text align='center'>
+                    <Localize i18n_default_text='Please use live chat to contact our Customer Support team for help.' />
+                </Text>
+            ),
+            icon: <DerivLightIcCashierBlockedIcon height={iconSize} width={iconSize} />,
+            title: (
+                <Text weight='bold'>
+                    <Localize i18n_default_text='Your Deriv P2P cashier is blocked' />
+                </Text>
+            ),
+        },
+        systemMaintenance: {
+            description: (
+                <div className='flex flex-col'>
+                    <Text align='center'>
+                        <Localize i18n_default_text='Please check back in a few minutes.' />
+                    </Text>
+                    <Text align='center'>
+                        <Localize i18n_default_text='Thank you for your patience.' />
+                    </Text>
+                </div>
+            ),
+            icon: <DerivLightIcCashierUnderMaintenanceIcon height={iconSize} width={iconSize} />,
+            title: (
+                <Text weight='bold'>
+                    <Localize i18n_default_text='Cashier is currently down for maintenance' />
+                </Text>
+            ),
+        },
     };
+
     return (
         <div className='pt-[2.4rem] m-[2.4rem]'>
-            {type && (
+            {type ? (
                 <ActionScreen
                     actionButtons={blockedScenarios[type].actionButton}
                     description={blockedScenarios[type].description}
                     icon={blockedScenarios[type].icon}
                     title={blockedScenarios[type].title}
                 />
+            ) : (
+                <ActionScreen
+                    actionButtons={blockedScenarios.p2pBlocked.actionButton}
+                    description={blockedScenarios.p2pBlocked.description}
+                    icon={blockedScenarios.p2pBlocked.icon}
+                    title={blockedScenarios.p2pBlocked.title}
+                />
```

Comment:
by default in v1, we show this if permission denied happens, so I have this return as the default blocked screen when user gets permission denied

Reply:
why not give type='p2pBlocked' as default?

Reply:
let me try this 👀